config/nav.yml

```diff
@@ -95,6 +95,21 @@ nav:
     - Serving:
       - Knative Serving overview: serving/README.md
       - Resources:
+        - Services:
+          - About Knative Services: serving/services/README.md
+          - Creating a Service: serving/services/creating-services.md
+          - Configuring Services:
+            - Configuring private Services: serving/services/private-services.md
+            - Permanent data storage: serving/services/pvc.md
```

Comment:
I would recommend something like: "Data Storage Access" or "Storage Access", storage can be non permanent. 
In any case to be precise: "A PersistentVolumeClaim (PVC) is a request for storage by a user" that is why I am more inclined to change this.
Unless we are going to split storage to permanent and ephemeral.
For the latter we support [empty dir ]( https://deploy-preview-5507--knative.netlify.app/docs/serving/configuration/feature-flags/#kubernetes-emptydir-volume) which is part of the ephemeral storage volume types [provided by K8s](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#types-of-ephemeral-volumes).